views/index.handlebars

```diff
@@ -32,6 +29,7 @@
 	</tr>
 </table>
 <div class="button-row">
+	 {{>form}}
```

Comment:
Please remove this, we don't want to have the form on the main homepage. It should only be visible on the add topic page and edit page